package.json

```diff
@@ -28,9 +28,9 @@
         "global:lint-check": "cd $INIT_CWD && eslint src/ --ext .js,.jsx,.ts,.tsx",
         "global:scss-lint": "echo 'TODO: Set this up.'",
         "global:lint-fix": "cd $INIT_CWD && eslint --fix src/ --ext .js,.jsx,.ts,.tsx",
-        "global:test": "cd $INIT_CWD && jest",
+        "global:test": "cd $INIT_CWD && NODE_ENV=test jest",
```

Comment:
Jest normally does this by default, but our docker container already specifies NODE_ENV=development.

This generally seems better, but the Immediate motivation is that MUI logs some console errors that are silenced (correctly) during tests. For example, MUI logs an error when tab panels have zero height / width. But in JSDOM-based test environments (like jest) the height will always be zero since JSDOM does not implement any layout logic. (In particular, it returns a bunch of zeros for `getBoundingClientRect`).